examples/quickstart/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     )
     set_property(
       TEST ${TEST_NAME}
-      APPEND PROPERTY DEPENDS simdjson-source ${PROJECT_SOURCE_DIR}/examples/quickstart/${SOURCE_FILE}
```

Comment:
This target is explicitly trying not to use simdjson-flags, which is why it includes simdjson-source and not simdjson-internal. The idea is for the quickstart to be as close to the quickstart instructions in README.md as possible--and those instructions don't pass any flags at all.